```
======================================================================
FAIL: test_instantiate_all (testapps.test_pillowtop.tests.test_settings.PillowtopSettingsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/testapps/test_pillowtop/tests/test_settings.py", line 30, in test_instantiate_all
    self.assertEqual(expected_meta[pillow_config.name], _pillow_meta_from_config(pillow_config))
AssertionError: {u'change_feed_type': u'KafkaChangeFeed', u'full_class_name': u'pillowtop.pillow [truncated]... != {'change_feed_type': 'KafkaChangeFeed', 'checkpoint_id': u'CaseToElasticsearchPi [truncated]...
- {u'advertised_name': u'CaseToElasticsearchPillow',
?  -                   -
+ {'advertised_name': 'CaseToElasticsearchPillow',
-  u'change_feed_type': u'KafkaChangeFeed',
?  -                    -
+  'change_feed_type': 'KafkaChangeFeed',
-  u'checkpoint_id': u'CaseToElasticsearchPillow-test_hqcases_2016-03-04',
?  -                                                             ^    ^^
+  'checkpoint_id': u'CaseToElasticsearchPillow-test_hqcases_2017-03-28',
?                                                               ^    ^^
-  u'full_class_name': u'pillowtop.pillow.interface.ConstructedPillow',
?  -                   -
+  'full_class_name': 'pillowtop.pillow.interface.ConstructedPillow',
-  u'name': u'CaseToElasticsearchPillow'}
?  -        -
+  'name': 'CaseToElasticsearchPillow'}
```

--- a/corehq/pillows/mappings/case_mapping.py
+++ b/corehq/pillows/mappings/case_mapping.py
@@ -1,10 +1,11 @@
+from copy import deepcopy
 from corehq.pillows.base import DEFAULT_META
 from corehq.pillows.core import DATE_FORMATS_ARR, DATE_FORMATS_STRING
 from corehq.pillows.mappings import NULL_VALUE
 from corehq.util.elastic import es_index
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-CASE_INDEX = es_index("hqcases_2016-03-04")
+CASE_INDEX = es_index("hqcases_2017-03-28")
 CASE_ES_TYPE = 'case'
 
 CASE_MAPPING = {
@@ -104,10 +105,17 @@ CASE_MAPPING = {
 
 CASE_ES_ALIAS = "hqcases"
 
+CASE_INDEX_META = deepcopy(DEFAULT_META)
+CASE_INDEX_META.update({
+    'settings': {
+        'number_of_shards': 10,
+    },
+})
+
 CASE_INDEX_INFO = ElasticsearchIndexInfo(
     index=CASE_INDEX,
     alias=CASE_ES_ALIAS,
     type=CASE_ES_TYPE,
-    meta=DEFAULT_META,
+    meta=CASE_INDEX_META,
     mapping=CASE_MAPPING
 )

--- a/corehq/pillows/mappings/case_mapping.py
+++ b/corehq/pillows/mappings/case_mapping.py
@@ -1,4 +1,5 @@
 from copy import deepcopy
+from django.conf import settings
 from corehq.pillows.base import DEFAULT_META
 from corehq.pillows.core import DATE_FORMATS_ARR, DATE_FORMATS_STRING
 from corehq.pillows.mappings import NULL_VALUE
@@ -106,11 +107,13 @@ CASE_MAPPING = {
 CASE_ES_ALIAS = "hqcases"
 
 CASE_INDEX_META = deepcopy(DEFAULT_META)
-CASE_INDEX_META.update({
-    'settings': {
-        'number_of_shards': 10,
-    },
-})
+
+if settings.SERVER_ENVIRONMENT == 'production':
+    CASE_INDEX_META.update({
+        'settings': {
+            'number_of_shards': 10,
+        },
+    })
 
 CASE_INDEX_INFO = ElasticsearchIndexInfo(
     index=CASE_INDEX,

--- a/corehq/pillows/mappings/xform_mapping.py
+++ b/corehq/pillows/mappings/xform_mapping.py
@@ -1,10 +1,11 @@
+from copy import deepcopy
 from corehq.pillows.base import DEFAULT_META
 from corehq.pillows.core import DATE_FORMATS_STRING, DATE_FORMATS_ARR
 from corehq.pillows.mappings import NULL_VALUE
 from corehq.util.elastic import es_index
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-XFORM_INDEX = es_index("xforms_2016-07-07")
+XFORM_INDEX = es_index("xforms_2017-03-28")
 
 XFORM_MAPPING = {
     "date_detection": False,
@@ -120,11 +121,17 @@ XFORM_MAPPING = {
 
 XFORM_ES_TYPE = 'xform'
 XFORM_ALIAS = "xforms"
+XFORM_INDEX_META = deepcopy(DEFAULT_META)
+XFORM_INDEX_META.update({
+    'settings': {
+        'number_of_shards': 10,
+    },
+})
 
 XFORM_INDEX_INFO = ElasticsearchIndexInfo(
     index=XFORM_INDEX,
     alias=XFORM_ALIAS,
     type=XFORM_ES_TYPE,
-    meta=DEFAULT_META,
+    meta=XFORM_INDEX_META,
     mapping=XFORM_MAPPING,
 )

--- a/corehq/pillows/mappings/xform_mapping.py
+++ b/corehq/pillows/mappings/xform_mapping.py
@@ -1,4 +1,5 @@
 from copy import deepcopy
+from django.conf import settings
 from corehq.pillows.base import DEFAULT_META
 from corehq.pillows.core import DATE_FORMATS_STRING, DATE_FORMATS_ARR
 from corehq.pillows.mappings import NULL_VALUE
@@ -122,11 +123,13 @@ XFORM_MAPPING = {
 XFORM_ES_TYPE = 'xform'
 XFORM_ALIAS = "xforms"
 XFORM_INDEX_META = deepcopy(DEFAULT_META)
-XFORM_INDEX_META.update({
-    'settings': {
-        'number_of_shards': 10,
-    },
-})
+
+if settings.SERVER_ENVIRONMENT == 'production':
+    XFORM_INDEX_META.update({
+        'settings': {
+            'number_of_shards': 10,
+        },
+    })
 
 XFORM_INDEX_INFO = ElasticsearchIndexInfo(
     index=XFORM_INDEX,

--- a/testapps/test_elasticsearch/tests/test_prod_es_indices.py
+++ b/testapps/test_elasticsearch/tests/test_prod_es_indices.py
@@ -75,8 +75,8 @@ EXPECTED_PROD_INDICES = [
     },
     {
         "alias": "hqcases",
-        "index": "test_hqcases_2016-03-04",
-        "type": "case",
+        "index": "test_hqcases_2017-03-28",
+        "type": "case"
         "meta": {
             "settings": {
                 "analysis": {
@@ -277,8 +277,8 @@ EXPECTED_PROD_INDICES = [
     },
     {
         "alias": "xforms",
-        "index": "test_xforms_2016-07-07",
-        "type": "xform",
+        "index": "test_xforms_2017-03-28",
+        "type": "xform"
         "meta": {
             "settings": {
                 "analysis": {

